services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/manage_units/manage_units.jsx

```diff
@@ -26,6 +26,7 @@ export default class ManageUnits extends React.Component {
       classrooms: [],
       selectedClassroomId: getParameterByName('classroom_id') || allClassroomKey,
       activityWithRecommendationsIds: [],
+      open: !window.location.pathname.includes('closed')
```

Comment:
This feels like it could be a little bit brittle.  Is there a way to get an attribute passed in through `ReactRouter`?  Maybe something like `useParams()`?  Not sure if that'd work with our current routing.